tests/pr_test.go

```diff
@@ -128,8 +128,10 @@ func TestRunDASolutionSchematics(t *testing.T) {
 
 	options.TerraformVars = []testschematic.TestSchematicTerraformVar{
 		{Name: "ibmcloud_api_key", Value: options.RequiredEnvironmentVars["TF_VAR_ibmcloud_api_key"], DataType: "string", Secure: true},
+		{Name: "prefix", Value: options.Prefix, DataType: "string"},
 		{Name: "region", Value: options.Region, DataType: "string"},
-		{Name: "resource_group_name", Value: options.Prefix, DataType: "string"},
+		{Name: "existing_resource_group", Value: true, DataType: "string"},
```

Comment:
remove this - we want the test to test creating a new resource group using prefix

Reply:
Shall I remove this assignment as well from the `TestRunDASolutionSchematics` -
`ResourceGroup:          resourceGroup,`

Reply:
No this indicates the resource group that the schematics workspace will be created in, so its needed for that